src/main/charts/jira/Changelog.md

```diff
@@ -1,13 +1,37 @@
 # Change Log
 
+## 1.1.0 
+
+**Release date:** 2021-11-03
+
+![AppVersion: 8.20.1-jdk11](https://img.shields.io/static/v1?label=AppVersion&message=8.20.1-jdk11&color=success&logo=)
+![Kubernetes: >=1.19.x-0](https://img.shields.io/static/v1?label=Kubernetes&message=>=1.19.x-0&color=informational&logo=kubernetes)
+![Helm: v3](https://img.shields.io/static/v1?label=Helm&message=v3&color=informational&logo=helm)
+
+
+* DCKUBE-721: Update version in Chart.yaml files 
+* DCKUBE-733: Update the product versions (#345) 
+* DCD-1380: Update Jira to 8.20 LTS (#339) 
+* DCKUBE-739: Fix typos (#337) 
+* DCKUBE-739: make securityContext changes backward compatible (#332) 
+* Roll Statefulset Pods if ConfigMap changes (#315) 
```

Comment:
To be consistent can we have the ticket number for this change as well?

Reply:
There was no ticket, as this was a community contribution